app-misc/prog-express/prog-express-3.7.8.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils multilib
```

Comment:
Any explanation why are you using `multilib.eclass` here? I guess you did this to provide `$(get_libdir)` feature, but this is built-in function for EAPI=6

Reply:
You are right, I wasn't aware, that this is part if EAPI=6. Will be fixed.